File: web_app/duta/Interface/LoginResponse.cs
namespace duta.Interface
{
    public class LoginResponse
    {
        public LoginResponse(bool status, int user_id)
        {
            logged_in = status ? 1 : 0;
            this.user_id = user_id;
        }

        public int logged_in { get; set; }
        public int user_id { get; set; }
    }
}

File: android/Duta/DutaMobile/src/main/java/com/dutamobile/model/response/LoginResponse.java
package com.dutamobile.model.response;

/**
 * Created by Bartosz on 15.11.13.
 */
public class LoginResponse
{
    public int isLoggedIn()
    {
        return logged_in;
    }

    public void setLoggedIn(int logged_in)
    {
        this.logged_in = logged_in;
    }

    private int logged_in;

    public int getUser_id()
    {
        return user_id;
    }

    public void setUser_id(int user_id)
    {
        this.user_id = user_id;
    }

    private int user_id;
}
